src/config/mongoose_config_spec.erl

```diff
@@ -646,6 +647,17 @@ auth_rdbms() ->
        format = none
       }.
 
+%% path: (host_config[].)auth.dummy
+auth_dummy() ->
```

Comment:
how about to test it? e.g. we can set this configuration options for "test type" host_type.

Reply:
Pushed so :)

Reply:
Here, now carboncopy for dynamic domains is pretty fast: https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3131/55777/pgsql_mnesia.23.0.3-1/big/ct_run.test@default-44683ef9-9b3c-4852-b093-e90d801f50e2.2021-05-27_09.27.12/big_tests.tests.carboncopy_SUITE.logs/run.2021-05-27_09.27.14/suite.log.html 😄